exporter/exporterhelper/internal/persistent_storage_batch.go

```diff
@@ -59,15 +59,16 @@ func (bof *batchStruct) execute(ctx context.Context) (*batchStruct, error) {
 }
 
 // set adds a Set operation to the batch
-func (bof *batchStruct) set(key string, value interface{}, marshal func(interface{}) ([]byte, error)) *batchStruct {
```

Comment:
Instead of making all these functions explicitly return the size, can't we just add a method to `batchStruct` that would calculate the total data size? Data in a batchstruct is already marshalled, so it'd be just iterating over operations.